### DISCUSSION TITLE
Add different post view options

### DISCUSSION THREAD

**Discussion by ToNIX-** [aeharding/voyager#90] [Ideas]
It would be amazing to have multiple options to view posts, such as cards, small cards and list. Would it be possible since it's a web app?

Thank you!

**Comment by aeharding** [comment]
Hi there!

Yes, there's a plan for compact view: #65

apart from that, I changed this to a discussion so we can further talk about other options for the feed :)

**Comment by ToNIX-** [comment]
Ah, I'm sorry I missed it. I searched for compact, cards and different keywords, but totally missed #65.
